pidgin/app.py

```diff
@@ -111,11 +110,15 @@ def send_query(query_txt):
 
     auth = flask.request.headers.get('Authorization')
     query = {'query': query_txt}
-    data = requests.post(api_url, headers={'Authorization': auth}, json=query).json()
+    response = requests.post(api_url, headers={'Authorization': auth}, json=query)
+    data = response.json()
+
+    if response.status_code == 403:
+        raise AuthenticationException(data['message'])
+
+    return data
```

Comment:
I think this still needs `flask.jsonify(data)`

Reply:
Removing `flask.jsonify()` allows me to remove `.get_json()` later, i thought it would be more readable

Reply:
Yeah, that comment doesn't make sense, I misread